modelforge/potential/processing.py

```diff
@@ -0,0 +1,339 @@
+import torch
+from modelforge.dataset.dataset import DatasetStatistics
+from loguru import logger as log
+
+
+class FromAtomToMoleculeReduction(torch.nn.Module):
```

Comment:
While this function is generic in what it does (summing the per atom quantities) all the variable names and description refer to energy (obviously this is what we are using it for in the code).  Not sure if it makes sense to rename the class to like "FromAtomToMoleculeEnergyReduction".  Obviously this is minor, but given the ever increasing complication of the code, being more specific could be better.

Reply:
I think it might be better to move in the opposite direction and remove all references to `energy`. It should be generic, then we don't need to add another class to perform the same calculation for some other property.